src/controllers/user.ts

```diff
@@ -1,21 +1,54 @@
 import { Request, Response } from 'express'
 import { body, check, validationResult } from 'express-validator'
+import { User } from '../entity'
+import { logger } from '../logger'
 
-export const getAccount = (req: Request, res: Response): Response => {
-  if (req.user) {
-    return res.json(req.user.toJSON())
+export const getAccount = async (req: Request, res: Response): Promise<Response> => {
+  const user = req.user
+
+  if (!user) {
+    return res.status(401).json({ error: 'not logged in.' })
   }
-  return res.status(501).json({ error: 'not implemented' })
+
+  return res.status(200).json({ user })
```

Comment:
I'm going to update this to use `toJSON` to be safe and to also allow all keys to be returned:
```suggestion
  return res.json({ user: user.toJSON() })
```
Also we can remove status `200` since it is inferred, only need to call `status` when it's anything but `200`.